rudder-lang/libs/oslib.rl

```diff
@@ -39,16 +65,17 @@ items in bsd {
 }
 
 items in linux {
-  alpine_linux,
-  arch_linux,
+# TODO do we keep android ?
```

Comment:
I don't think it really matters as it is not a list of supported OS. I'm fine with both.

Reply:
let's go with a no, since this list is extensible